lib/slack/block_kit/element/checkbox_group.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Slack
+  module BlockKit
+    module Element
+      # A checkbox group that allows a user to choose multiple items from
+      # a list of possible options.
+      #
+      # https://api.slack.com/reference/messaging/block-elements#checkboxes
+      class CheckboxGroup
```

Comment:
Not sure if this should be `CheckboxGroup` or `Checkboxes` to be consistent with the `TYPE`.

Reply:
I think `Checkboxes` makes sense 👍